src/coreComponents/physicsSolvers/contact/LagrangianContactSolver.cpp

```diff
@@ -65,33 +64,28 @@ LagrangianContactSolver::LagrangianContactSolver( const string & name,
     setInputFlag( InputFlags::REQUIRED ).
     setDescription( "Name of the stabilization to use in the lagrangian contact solver" );
 
-  m_linearSolverParameters.get().mgr.strategy = LinearSolverParameters::MGR::StrategyType::lagrangianContactMechanics;
-  m_linearSolverParameters.get().mgr.separateComponents = false;
-  m_linearSolverParameters.get().mgr.displacementFieldName = solidMechanics::totalDisplacement::key();
-  m_linearSolverParameters.get().dofsPerNode = 3;
+  LinearSolverParameters & linSolParams = m_linearSolverParameters.get();
+  linSolParams.mgr.strategy = LinearSolverParameters::MGR::StrategyType::lagrangianContactMechanics;
+  linSolParams.mgr.separateComponents = true;
```

Comment:
I don't think this can be applied since we have AMG as a coarse solver only. Not too sure what the effect of this would be.

Reply:
these calls should overwrite whatever the parent class assigns though. I think I had added an integratedTest using MGR for the embedded case. If that one works I think we should be okay.

Reply:
just to document:

LagrangianContactSolver (child)
```
  linSolParams.mgr.strategy = LinearSolverParameters::MGR::StrategyType::lagrangianContactMechanics;
  linSolParams.mgr.separateComponents = true;
  linSolParams.mgr.displacementFieldName = solidMechanics::totalDisplacement::key();
  linSolParams.dofsPerNode = 3;
```

SolidMechanicsLagrangianFEM (parent)
```
  linParams.isSymmetric = true;
  linParams.dofsPerNode = 3;
  linParams.amg.separateComponents = true;
```

it does not seem to affect the results:
```
Rank 0 is comparing /tmp/geos-build/integratedTests/workingDir/poromechanicsFractures/PoroElastic_efem-edfm_inclinedFrac_smoke_01/0to1_restart_000000010/rank_0000000.hdf5 with /tmp/geos/integratedTests/poromechanicsFractures/PoroElastic_efem-edfm_inclinedFrac_smoke_01/0to1_restart_000000010/rank_0000000.hdf5 
...
********************************************************************************
Error: /Problem/Solvers/fractureMechSolver/LinearSolverParameters/amgSeparateComponents/__values__
	Scalar values of types int32 and int32 differ: 1, 0.
```
but no solution diffs are detected

Reply:
Yeah, I don't think it does anything if it's not the first MGR level F-relaxation.